pkgs/servers/matrix-synapse/matrix-appservice-irc/update.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env nix-shell
-#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix
+#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix nix-prefetch-git
 
 set -euo pipefail
 # cd to the folder containing this script
 cd "$(dirname "$0")"
 
-CURRENT_VERSION=$(nix eval --raw '(with import ../../../../. {}; matrix-appservice-irc.version)')
+CURRENT_VERSION=$(nix eval -f ../../../../. --raw matrix-appservice-irc.version)
```

Comment:
Can we use the stable CLI instead?

Reply:
I'm not sure what the equivalent from the 2.3 CLI would be. But I agree this can be problematic for users that didn't opt in to the experimental commands. *sigh*